docs/modules/ROOT/pages/servlet/oauth2/client/core.adoc

```diff
@@ -69,7 +69,8 @@ This information is available only if the Spring Boot property `spring.security.
 <15> `(userInfoEndpoint)authenticationMethod`: The authentication method used when sending the access token to the UserInfo Endpoint.
 The supported values are *header*, *form*, and *query*.
 <16> `userNameAttributeName`: The name of the attribute returned in the UserInfo Response that references the Name or Identifier of the end-user.
-<17> [[oauth2Client-client-registration-requireProofKey]]`requireProofKey`: If `true` or if `authorizationGrantType` is `none`, then PKCE will be enabled by default.
+<17> [[oauth2Client-client-registration-requireProofKey]]`requireProofKey`: This will by default be `true`, as PKCE will be enabled by default.
```

Comment:
Please update the text to:

If `true` or if `clientAuthenticationMethod` is `none`, then PKCE will be enabled.

Reply:
Aren't we stressing enough on the point that PKCE will be also enabled for confidential clients?... Or we don't want to for now